instrumentation/pg/test/opentelemetry/instrumentation/pg/instrumentation_test.rb

```diff
@@ -12,13 +12,16 @@
 require_relative '../../../../lib/opentelemetry/instrumentation/pg/patches/connection'
 
 # This test suite requires a running postgres container and dedicated test container
-# To run tests:
+# To run tests locally:
 # 1. Build the opentelemetry/opentelemetry-ruby-contrib image
 # - docker-compose build
 # 2. Bundle install
 # - docker-compose run ex-instrumentation-pg-test bundle install
-# 3. Run test suite
-# - docker-compose run ex-instrumentation-pg-test bundle exec rake test
+# 3. Install the Appraisal gem (https://github.com/thoughtbot/appraisal)
```

Comment:
The apprasial gem should already be installed from step 2, maybe something like: "Install the dependencies for each Appraisal" or "Install the Appraisals"? I like your link to the gem :)

Reply:
Yes that is better!